src/Controls/tests/TestCases.HostApp/Issues/Issue23411.xaml.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.PlatformConfiguration;
+using Microsoft.Maui.Controls.PlatformConfiguration.iOSSpecific;
+using Application = Microsoft.Maui.Controls.Application;
+using NavigationPage = Microsoft.Maui.Controls.NavigationPage;
+
+namespace Maui.Controls.Sample.Issues
+{
+	[Issue(IssueTracker.Github, 23411, "[iOS]Use non-overridden traits in AppInfoImplementation.RequestTheme",
+		PlatformAffected.iOS)]
+   public partial class Issue23411 : ContentPage
+    {
+        public Issue23411()
+        {
+            InitializeComponent();
+        }
+ 
+        protected override async void OnAppearing()
+        {
+            Application.Current.UserAppTheme = AppTheme.Dark;
+            await Task.Delay(500);
```

Comment:
Do we need this here?

Reply:
Hi @PureWeen , Previously, we encountered a random issue on CI where screenshots were captured before the theme change was applied. To address this, await was added. The code changes have now been removed, and it works fine locally.